src/amo/components/GetFirefoxButton/index.js

```diff
@@ -193,6 +195,7 @@ export const GetFirefoxButtonBase = ({
         className,
       )}
       href={`${DOWNLOAD_FIREFOX_BASE_URL}${makeQueryStringWithUTM({
+        ...overrideQueryParams,
```

Comment:
This should be added last, otherwise it won't override all parameters (I need to override `utm_term` for the blog). I just found out while updating #10523